<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>315</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>315</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>370</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="closeDialog">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="general">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>186</height>
       </rect>
      </property>
      <attribute name="label">
       <string>General settings</string>
      </attribute>
      <attribute name="toolTip">
       <string>General, uncategorised settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Database file:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dbfile"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Startup language:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="polishLang">
            <property name="text">
             <string notr="true">Polski</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="germanLang">
            <property name="text">
             <string notr="true">Deutsch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="englishLang">
            <property name="text">
             <string notr="true">English</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="playlist">
      <attribute name="label">
       <string>Playlist columns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Available</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QListWidget" name="plAvail">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>141</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QListWidget" name="plVisible">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>141</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
